### DISCUSSION TITLE
Website banner with pandoc theme?

### DISCUSSION THREAD

**Discussion by austin-hoover** [quarto-dev/quarto-cli#11333] [Q&A] [answered]
### Description

I like the pandoc website theme, but it removes the top banner from my website. I'd like to at least have a link back to the homepage. Has anyone found a solution?

**Comment by mcanouil** [comment]
Sorry but it's not very clear what your issue is and what is your need.
Also what do you mean with "Pandoc" here?

Did you look at the documentation? https://quarto.org/docs/websites/


Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by cderv** [comment, marked as answer]
> I like the pandoc website theme

Should we assume you are referring to `theme: pandoc` ? 

If so let me add a precision on this: Setting this for HTML format means that no bootstrap will be used. 
Most of the website feature are based on Bootstrap, like navbar or sidebar elements. So no navigation will be added for such website. 

I believe in this case, everything needs to be done manually if you want navigation to another page or a button to home page. You could add a backlink by modifying pandoc's template for html I think, as we do insert Body envelope for navigation, for boostrap pages.
